docs/source/release_notes/release-notes.rst

```diff
@@ -45,6 +45,11 @@ develop
          - Change
 
     *    - |improved|
+         - Sweep progress is now persisted as a blob. This allows us to use check and set to avoid versioning the entries in the sweep progress table.
+           No migration is necessary as the data is persisted to a new table, however sweep will start sweeping a new table since the previously persisted sweep progress will be ignored.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2555>`__)
```

Comment:
Might be worth mentioning that previously, the entries were cleared out only when a table was fully swept, leading to O(n^2) (I think?) performance when sweeping a very large table.

Reply:
Could be a user break - what happens if they upgrade while in the middle of a large, high-priority-for-sweeping table?